eng/pipelines/build-windows-job.yml

```diff
@@ -42,6 +42,12 @@ jobs:
   steps:
     - template: checkout-windows-task.yml
 
+    - task: NuGetAuthenticate@1
+      displayName: 🔏 Authenticate NuGet feeds
+      inputs:
+        nuGetServiceConnections: azure-public/roslyn
```

Comment:
I dont seem to have access to view this actual service connection though to validate.